tools/debug_tools/src/lib.rs

```diff
@@ -41,8 +39,7 @@ pub use bevy_inspector_egui::bevy_egui;
 pub struct DebugInfo {
     /// Toggle global access to developer tools
     pub dev_mode: bool,
-    // TODO: see: https://github.com/Leafwing-Studios/Emergence/issues/140
-    // /// Toggle developer console
+    /// Toggle developer console
```

Comment:
Should we still keep this TODO here?

Reply:
I tried to remove these where they were.